.gitattributes

```diff
@@ -11,3 +11,5 @@
 *.y text
 demo/primer.css binary
 demo/libwabt.js binary
+demo/third_party/codemirror/codemirror.css binary
+demo/third_party/codemirror/codemirror.js binary
```

Comment:
But I read it on the internet! :-)
https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes